pvlib/spectrum/mismatch.py

```diff
@@ -358,21 +358,17 @@ def integrate(e):
 def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
                                module_type=None, coefficients=None,
                                min_precipitable_water=0.1,
-                               max_precipitable_water=8):
+                               max_precipitable_water=8,
+                               min_airmass_absolute=0.58,
```

Comment:
```suggestion
                               min_airmass_absolute=1.0,
```
This value should be justified by the model development or validation rather than by geography.  The model was developed using spectra generated for air mass values between 1 and 5 at sea level, and I have no indications that it was validated for air mass values < 1.0.

Reply:
Glad you picked up on this. I was not sure if that was also part of the extrapolation you mentioned [here](https://github.com/pvlib/pvlib-python/issues/2086#issuecomment-2182910664) and was discussed more broadly in #2086, so just left the default 0.58 as it was in the original script, but I should have checked. Thanks for picking up on that.

Reply:
> Where does the default value come from (a quote or screen-shot from the paper perhaps)?

@AdamRJensen the default 0.58 in the original script was not in the paper. Screenshot [here](https://github.com/pvlib/pvlib-python/issues/2086#issuecomment-2181226255) shows model development limits between 1 and 5. Original script defaulted to  0.58, not 1.0; I suggested including a parameter to enable the user to modify this value if they wish. Here, adriesse suggests defaulting to 1.0.

Reply:
> Glad you picked up on this @adriesse. I was not sure if that was also part of the extrapolation you mentioned [here](https://github.com/pvlib/pvlib-python/issues/2086#issuecomment-2182910664) and was discussed more broadly in #2086, so just left the default 0.58 as it was in the original script, but I should have checked. Thanks for picking up on that.

Actually I hadn't noticed it was in the code before.  This extrapolation could possibly be considered as part of the model, but we have no evidence of validation data in that range.

Reply:
I haven't seen any published evidence for validity specifically within the default range either...
If we are retaining limits in the function then since these are the defaults set by the original author from FS, perhaps we leave them as they are for now?

Reply:
Yes, probably better to leave the historical flaws in place.